packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -266,19 +266,26 @@ const TESTS: [string, any, Record<string, any>?][] = [
       ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.lowRecoveryCodes }],
     ])],
     ['html', [
+      { test: 'include', expected: '2 recovery codes remaining' },
       { test: 'include', expected: decodeUrl(configHref('accountRecoveryCodesUrl', 'low-recovery-codes', 'recovery-codes', 'low_recovery_codes=true', 'email', 'uid')) },
       { test: 'include', expected: decodeUrl(configHref('privacyUrl', 'low-recovery-codes', 'privacy')) },
       { test: 'include', expected: decodeUrl(configHref('supportUrl', 'low-recovery-codes', 'support')) },
       { test: 'include', expected: 'alt="Firefox logo"' },
       { test: 'notInclude', expected: 'utm_source=email' },
     ]],
     ['text', [
+      { test: 'include', expected: 'Low recovery codes remaining' },
```

Comment:
Why is the text 'Low' and html '2'?

Reply:
"Low recovery codes remaining" is what we have set as the "title", which we apparently show sometimes in plaintext instead of the subject. I actually noted this inconsistency in another doc that's being reviewed by a content strategist, so hopefully this inconsistency will actually get resolved soon enough.